### PR TITLE
Various changes per latest beta

### PR DESCRIPTION
**Pull request by OzGav**

### CODE REVIEW

File: docs/music-providers/filesystem.md [hunk 1]
```diff
@@ -28,11 +28,15 @@ Music Assistant has support for SMB (also known as samba or CIFS) shares and DFS
...
-- Artwork which is downloaded is throttled to approximately one request every 30 seconds so it will take some time to download all required artwork with large collections. You can force the download by selecting "Refresh Item" from the ⋮ menu in the banner at the top of a view.
-- Embedded album thumbs will be extracted from audio files. However, you can save disk space by providing a single local artwork file vice embedding the same artwork in all files.
 - Use the following naming convention for local artwork (png should also work). Artist thumb: artist.jpg; Album thumb: folder.jpg or cover.jpg; Fan Art (used as background in banners): fanart.jpg. Artist thumb and Fanart should be in the folder with the artist name. Album thumbs should be in the folder with the album name or in the disc folders below that. More about artwork file types can be found here https://kodi.wiki/view/Artwork_types
+- Embedded album thumbs will be extracted from audio files. However, you can improve performance and save disk space by providing a single local artwork file vice embedding the same artwork in all files.
```
Comment (jozefKruszynski) [inline docs/music-providers/filesystem.md:35]:
```suggestion
- Embedded album thumbs will be extracted from audio files. However, you can improve performance and save disk space by providing a single local artwork file vs. embedding the same artwork in all files.
```

File: docs/ui.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ The behaviour described above will work on any view where you see the "x item(s)
 
 ![image](assets/screenshots/UI-individual-artist.png)
 
-The tracks and albums section are collapsible. The view above shows them both collapsed. The tracks section shows ALL tracks. To see what albums any given track is on you can either sort alphabetically or click on one and use the context menu and SHOW INFO. 
+The tracks and albums section are collapsible. The view above shows them both collapsed. The tracks section shows ALL tracks that are in the library or that are available across all of the providers. A red dot on the library icon ![image](assets/icons/library-icon.png) indicates that the viewed results are filtered to only show items that are in the library. To see what albums any given track is on you can either sort alphabetically or click on one and use the context menu and SHOW INFO.
```
Comment (jozefKruszynski) [inline docs/ui.md:82]:
```suggestion
The tracks and albums sections are collapsible. The view above shows them both collapsed. The tracks section shows ALL tracks that are in the library or that are available across all of the providers. A red dot on the library icon ![image](assets/icons/library-icon.png) indicates that the viewed results are filtered to only show items that are in the library. To see what albums any given track is on you can either sort alphabetically or click on one and use the context menu and SHOW INFO. 
```

**Review by jozefKruszynski — CHANGES_REQUESTED** [review]
Couple of minimal changes that I think might make sense, but I could be wrong.

File: docs/music-providers/filesystem.md (hunk 1 above)
Comment (OzGav) [reply docs/music-providers/filesystem.md:35]: LGTM

**Review by jozefKruszynski — APPROVED** [review]

**Pull request merged**